app/frontend/src/components/admin/Clients.vue

```diff
@@ -12,20 +14,62 @@
           hide-details
           class="pb-5"
         ></v-text-field>
-      </v-card-title>
+      </div>
 
       <v-alert v-if="showAlert" :type="alertType" tile dense>{{alertMessage}}</v-alert>
 
+      <!-- table header -->
       <v-data-table
         class="kc-table"
-        dense
         :headers="headers"
         :items="serviceClients"
-        :items-per-page="5"
+        :items-per-page="10"
         :search="search"
         :loading="loading"
         loading-text="Loading... Please wait"
+        :expanded.sync="expanded"
+        item-key="acronym"
+        show-expand
       >
+        <template v-slot:expanded-item="{ headers, item }">
+          <td :colspan="headers.length">
+            <table class="kc-nested-table">
+              <!-- environments -->
+              <tr>
+                <td width="40px"></td>
```

Comment:
I'm not sure using a table inside this table row is going to work out (I might be wrong...). Especially with set pixel widths, as the column sizes for the parent table will vary with screen size. See my attached screenshot in a comment on this pr.

Reply:
yeah. i'll do something with divs